services/application.js

```diff
@@ -418,13 +419,12 @@ class Application {
         }
         const userInfo = context?.userInfo;
         const userScope = await this.#getUserScope(context?.userInfo, USER_PERMISSION_CONSTANTS.SUBMISSION_REQUEST.CANCEL);
-        if (userScope.isNoneScope() || !(userScope.isOwnScope() || userScope.isAllScope())) {
+        if (userScope.isNoneScope() || !(userScope.isOwnScope() && userScope.isAllScope())) {
```

Comment:
Following part doesn't look right to me. I assume user will never have own scope and all scope at the same time, so inside the parenthesis is always false?
`!(userScope.isOwnScope() && userScope.isAllScope())`